Add the 0BSD license

goneall opened the pull request:
Add the 0BSD license which is present on the [OSI website](https://opensource.org/licenses/0BSD) but not present in the repo.

Partially resolves issue #62 

Signed-off-by: Gary O'Neall <gary@sourceauditor.com>

goneall commented:
@webmink please review.  This was generated using a [python script](https://github.com/goneall/licenses/blob/importscript/licenses/spdx/importOsiApprovedFromSpdx.py) pulling the data from the SPDX JSON metadata.

If you see any issues, let me know and I will update the script before attacking the other missing licenses.

webmink commented:
@paultag @zacchiro First test case!  How would you like us all to handle this?

zacchiro commented:
I think the process should be decided by whoever is in charge of stewarding the OSI license approval process (independently from the API). Rationale: we shouldn't risk stray PR approvals that would make the API start announcing to the world that a license is OSI-approved when it isn't. For this part, I think requiring from licensing lead a comment in the relevant PR would be enough.

goneall commented:
@zacchiro @webmink Thanks for following up on the process.  I await this PR to be merged before proceeding with the other PR's.

One question - would you prefer me to submit a separate PR for each fix or would you like me to batch them into common issue (e.g. add missing SPDX ID's for existing licenses all in one PR)?

pchestek reviewed (approved):
Ok to add 0BSD to the repo.

webmink commented:
@paultag Please can you merge this one; I can do the ones that are all green but lack the experience with the ones that need prior actions.

goneall commented:
I added a commit to remove the first line of the text after comparing the plain text with the OSI website.

The plain text now matches the OSI website text.

goneall commented:
I ran into some validation errors on the 0BSD license - change to draft until it passes validation.

goneall commented:
OK - this now passes validation

goneall commented:
@pchestek Note that I made the following changes since your review:
- Removed the first line of the license text to match the OSI website
- Added a link to the OSI text

I should be finished making changes unless any of the reviews find additional issues.

goneall commented:
If the files from this PR is used to generate the website in the future, note that is is missing the "notes" text:
```
Note: Despite its name, Zero-Clause BSD is an alteration of the ISC license, and is not textually derived from licenses in the BSD family. Zero-Clause BSD was originally approved under the name "Free Public License 1.0.0".
```

If this text is supposed to be in the JSON file, please let me know which field(s) need to be updated so I can add those in the future.
